.justfile

```diff
@@ -12,7 +12,7 @@ test:
     ./gradlew test
 
 preitest := if path_exists('build/install/jbang/bin') != 'true' {
-  './gradlew installDist -x test'
+    './gradlew spotlessApply installDist -x test'
```

Comment:
why need to run spotlessapply again here?

Reply:
Not sure what you mean by "again". If you've not run the "build:" and want to run the "itests:" it has to do a build, right? It just seemed more consistent to have the same command as the "build:" action.